mkosi.extra/usr/lib/dracut/modules.d/01setup-ironic-python-agent/20-autologin-getty.conf

```diff
@@ -1,3 +0,0 @@
-[Service]
-ExecStart=
-ExecStart=-/sbin/agetty --skip-login --login-options root --noclear %I $TERM
```

Comment:
Why was the auto-login removed?

Reply:
That is now solved by the [config-option](https://github.com/sapcc/fedIPA/pull/21/files#diff-eecc9f3923757c5c1e627ce59942b0160198e2daa7ff4ab810d6d3cd5fae00a4R22).

Reply:
I see the config option. Very well. Simple failing is not good, of course.